web/src/pages/api/search.tsx

```diff
@@ -0,0 +1,119 @@
+import { PrismaClient } from "@prisma/client";
```

Comment:
This file should be named `.ts`. I noticed some people are doing `.tsx` on the backend, and that is not correct. I didn't catch it before, but I'm catching it now.